imports/api/coordinator/availabilitylog.js

```diff
@@ -21,7 +21,8 @@ Schema.AvailabilityLog = new SimpleSchema({
     optional: true,
   },
   affordances: {
```

Comment:
How specifically do you want to save the affordances? as your key pair values? e.g. {coffee:true, hour: 9}, etc.? would you ever have an key with a value of false?

also location log is the new availabilitylog (just something to be aware of when we merge you'll want to change this there)

Reply:
yes, key value pairs exactly as you said.  no, the key will never have a value of false.